.github/workflows/install_deps/action.yml

```diff
@@ -75,7 +75,7 @@ runs:
       if: inputs.install_tensorflow == 'true'
       env:
         TF_VERSION: ${{ inputs.tensorflow_version != '' && format('~={0}', inputs.tensorflow_version) || '' }}
-      run: pip install "tensorflow${{ env.TF_VERSION }}" "tf-keras${{ env.TF_VERSION }}"
+      run: pip install "tensorflow${{ env.TF_VERSION }}" "tf-keras${{ env.TF_VERSION }}"; pip freeze | grep "tensorflow"
```

Comment:
This was a temp change just to check that the correct version is installed. I will revert this before merging. Just waiting for CI to pass.